aspnetcore/fundamentals/servers/weblistener.md

```diff
@@ -0,0 +1,170 @@
+---
+title: WebListener server
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 10/27/2016
+ms.topic: article
+ms.assetid: 0a7286e4-6428-424e-b5c4-5c98815cf61c
+ms.prod: aspnet-core
+uid: fundamentals/servers/weblistener
+---
+# WebListener server for ASP.NET Core
+
+By [Tom Dykstra](http://github.com/tdykstra), [Chris Ross](https://github.com/Tratcher)
+
+WebListener is a [web server for ASP.NET Core](overview.md) that runs only on Windows. It's built on the [Http.Sys kernel mode driver](https://msdn.microsoft.com/en-us/library/windows/desktop/aa364510.aspx). WebListener is an alternative to [Kestrel](kestrel.md) that can be used for direct connection to the Internet without relying on IIS as a reverse proxy server. In fact, **WebListener can't be used with IIS, as it isn't compatible with the [ASP.NET Core Module](aspnet-core-module.md).**
+
+Although WebListener was developed for ASP.NET Core, it can be used directly in any .NET Core or .NET Framework application via the [Microsoft.Net.Http.Server](https://www.nuget.org/packages/Microsoft.Net.Http.Server/) NuGet package.
+
+WebListener supports the following features:
+
+- Windows Authentication 
+- Port sharing
+- HTTPS with SNI
+- HTTP/2 over TLS (Windows 10)
+- Direct file transmission
+- Response caching
+- WebSockets (Windows 8)
+
+Supported Windows versions:
+
+- Windows 7 and Windows Server 2008 R2 and later
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/servers/weblistener/sample)
+
+## When to use WebListener
+
+WebListener is primarily for deployments where you need to expose the server directly to the Internet without using IIS.
```

Comment:
"WebListener is useful for deployments...". It also exposes a number of Http.Sys features not available in Kestrel.